plugins/PagePerformance/javascripts/rowaction.js

```diff
@@ -33,20 +33,21 @@
 
     DataTable_RowActions_PagePerformance.prototype.performAction = function (label, tr, e, originalRow) {
         var apiMethod = this.dataTable.param.module + '.' + this.dataTable.param.action;
-        this.openPopover(apiMethod, label);
+        this.openPopover(apiMethod, label, this.dataTable.param.flat || 0);
     };
 
-    DataTable_RowActions_PagePerformance.prototype.openPopover = function (apiMethod, label) {
-        var urlParam = apiMethod + ':' + label;
+    DataTable_RowActions_PagePerformance.prototype.openPopover = function (apiMethod, label, isReportFlat) {
+        var urlParam = apiMethod + ':' + label + ':' + isReportFlat;
         DataTable_RowAction.prototype.openPopover.apply(this, [urlParam]);
     };
 
     DataTable_RowActions_PagePerformance.prototype.doOpenPopover = function (urlParam) {
         var urlParamParts = urlParam.split(':');
         var apiMethod = urlParamParts.shift();
         var label = decodeURIComponent(urlParamParts.shift());
+        var isReportFlat = urlParamParts.shift();
 
-        PagePerformance.show(apiMethod, label);
+        PagePerformance.show(apiMethod, label, isReportFlat);
```

Comment:
this parameter name confuses me `isReportFlat` 

I would expect a boolean but it seems you are passing an item from `urlParamParts` (whatever that is) or `undefined`.

Would casting it make it a bit clearer ?

Reply:
If I understand the change correctly I think it's a 0/1 int-encoded boolean as you can't easily type boolean in URLs.

Reply:
exactly